### PR TITLE
[READY FOR REVIEW] [Simplified Chinese] W3C Accessibility Standards Overview

### PR DESCRIPTION
**Pull request by heybran**
Resolves https://github.com/w3c/wai-translations/issues/255

### CODE REVIEW

**Review by JinMokai — APPROVED** [review]
@remibetin  The translation looks okay.🎉

File: pages/standards-guidelines/index.zh-hans.md [hunk 1]
```diff
@@ -132,7 +133,7 @@ ARIA 信息：
 
 下面的资源支持可访问性评估方法和工具的开发：
 
-- [[无障碍一致性测试（ACT）概述]](/standards-guidelines/act/) &mdash; ACT 建立并记录规则，用来测试 Web 内容是否与可访问性标准一致。
+- [[无障碍一致性测试（ACT）概述]](/standards-guidelines/act/) &mdash; ACT 建立并记录规则，用来测试 Web 内容是否与可访问性标准一致。[WCAG 2 测试规则](/standards-guidelines/act/rules/about) 使用ACF规则格式。
```
Comment (remibetin) [inline pages/standards-guidelines/index.zh-hans.md:136]:
```suggestion
- [[无障碍一致性测试（ACT）概述]](/standards-guidelines/act/) &mdash; ACT 建立并记录规则，用来测试 Web 内容是否与可访问性标准一致。[WCAG 2 测试规则](/standards-guidelines/act/rules/about) 使用ACT规则格式。
```

Typo?

**Review by remibetin — CHANGES_REQUESTED** [review]
Thanks @heybran. I have noticed a few things:
- The "Personalization" section needs update. It is now named "WAI-Adapt" and its content has been updated in 2022 (see the [Changes history](https://www.w3.org/WAI/standards-guidelines/changelog/#30-june-2022))
- Image: we have recently changed the way we link to the long description of the image. Can you apply the new method? The new code is available at: (please translate the alt text)
https://github.com/w3c/wai-website/blob/669c8821b3475907c6c322ddd94d053281454cdc/pages/standards-guidelines/index.md?plain=1#L171-L172
- If you are interested in translating the image, please read [How to Translate Images
](https://www.w3.org/WAI/about/translating/resources/images/). Let me know if you prefer we publish the translation without the translated image for now.

**Comment by heybran** [comment]
@remibetin Translations updated: https://deploy-preview-953--wai-website.netlify.app/standards-guidelines/zh-hans

Regarding:

> If you are interested in translating the image, please read [How to Translate Images](https://www.w3.org/WAI/about/translating/resources/images/). Let me know if you prefer we publish the translation without the translated image for now.

Please kindly publish the translation without the translated image for now if the updated translations are all good.

**Comment by remibetin** [comment]
Thanks @heybran. I am checking a few things and should be able to get back to you next week.

File: pages/standards-guidelines/index.zh-hans.md [hunk 2]
```diff
@@ -132,18 +135,17 @@ ARIA 信息：
 
 下面的资源支持可访问性评估方法和工具的开发：
 
-- [[无障碍一致性测试（ACT）概述]](/standards-guidelines/act/) &mdash; ACT 建立并记录规则，用来测试 Web 内容是否与可访问性标准一致。
+- [[无障碍一致性测试（ACT）概述]](/standards-guidelines/act/) &mdash; ACT 建立并记录规则，用来测试 Web 内容是否与可访问性标准一致。[WCAG 2 测试规则](/standards-guidelines/act/rules/about) 使用ACT规则格式。
```
Comment (remibetin) [inline pages/standards-guidelines/index.zh-hans.md:138]:
```suggestion
- [[无障碍一致性测试（ACT）概述]](/standards-guidelines/act/) &mdash; ACT 建立并记录规则，用来测试 Web 内容是否与可访问性标准一致。[WCAG 2 测试规则](/standards-guidelines/act/rules/about/) 使用ACT规则格式。
```

**Review by remibetin — CHANGES_REQUESTED** [review]

**Comment by remibetin** [comment]
Thanks @heybran. I have completed my review and added a few comments.

**Comment by heybran** [comment]
@remibetin Updated.

**Pull request merged**